<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCom</class>
 <widget class="QDialog" name="DlgCom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CANCEL">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_Stop">
      <property name="text">
       <string>停止位:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Check">
      <property name="text">
       <string>校验位:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Pote">
      <property name="text">
       <string>波特率:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Com">
      <property name="text">
       <string>串口号:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Data">
      <property name="text">
       <string>数据位:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_Com"/>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_Check">
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EVEN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MARK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBox_Stop">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_Pote">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox_Data">
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_CANCEL</sender>
   <signal>clicked()</signal>
   <receiver>DlgCom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
